res/res_pjsip_notify.c

```diff
@@ -115,6 +116,23 @@
 			</configObject>
 		</configFile>
 	</configInfo>
+	<application name="PJPSIPNotify" language="en_US">
+		<synopsis>
+			Send a NOTIFY to either an arbitrary URI, or inside a SIP dialog.
+		</synopsis>
+		<syntax>
+			<parameter name="to" required="false">
+				<para>Abritrary URI to which to send the NOTIFY.  If none is specified, send inside</para>
+				<para>the current SIPdialog.</para>
+			</parameter>
+			<parameter name="headers" required="true">
+				<para>Either a pre-configured option or a list of headers to send in the NOTIFY.</para>
+			</parameter>
```

Comment:
I think you need lots of examples including how to set headers vs content vs pre-configured in pjsip_notify.conf.

Reply:
added examples in extensions, pjsip and pjsip_notify.

Reply:
You need examples here so they show up with `core show application PJSIPNotify`.
How do you set the body vs headers?  You need to explain that any parameter other than "Content" becomes a SIP header while "Content" becomes the message body.

There's also a mention of Content-Type in the AMI description.  How does that fit in here?

Reply:
Isn't this now a duplicate of what's in dialplan_functions_doc.xml?